tensorboard/defs/web.bzl

```diff
@@ -14,259 +14,295 @@
+    manifests = depset([manifest], transitive = [manifests])
+    return dummy, manifests
 
 def _new_file(ctx, suffix):
-  return ctx.actions.declare_file("%s%s" % (ctx.label.name, suffix))
+    return ctx.actions.declare_file("%s%s" % (ctx.label.name, suffix))
 
 def _add_webpath(ctx, src, webpath, webpaths, new_webpaths, manifest_srcs):
-  if webpath in new_webpaths:
-    _fail(ctx, "multiple srcs within %s define the webpath %s " % (
-        ctx.label, webpath))
-  if webpath in webpaths.to_list():
-    _fail(ctx, "webpath %s was defined by %s when already defined by deps" % (
-        webpath, ctx.label))
-  new_webpaths.append(webpath)
-  manifest_srcs.append(struct(
-      path=src.path,
-      longpath=long_path(ctx, src),
-      webpath=webpath))
+    if webpath in new_webpaths:
+        _fail(ctx, "multiple srcs within %s define the webpath %s " % (
+            ctx.label,
+            webpath,
+        ))
+    if webpath in webpaths.to_list():
+        _fail(ctx, "webpath %s was defined by %s when already defined by deps" % (
+            webpath,
+            ctx.label,
+        ))
+    new_webpaths.append(webpath)
+    manifest_srcs.append(struct(
+        path = src.path,
+        longpath = long_path(ctx, src),
+        webpath = webpath,
+    ))
 
 def _fail(ctx, message):
-  if ctx.attr.suppress == ["*"]:
-    print(message)
-  else:
-    fail(message)
+    if ctx.attr.suppress == ["*"]:
+        print(message)
+    else:
+        fail(message)
 
 def _get_path_relative_to_package(artifact):
-  """Returns file path relative to the package that declared it."""
-  path = artifact.path
-  for prefix in (artifact.root.path,
-                 artifact.owner.workspace_root if artifact.owner else '',
-                 artifact.owner.package if artifact.owner else ''):
-    if prefix:
-      prefix = prefix + "/"
-      if not path.startswith(prefix):
-        fail("Path %s doesn't start with %s" % (path, prefix))
-      path = path[len(prefix):]
-  return path
+    """Returns file path relative to the package that declared it."""
+    path = artifact.path
+    for prefix in (
+        artifact.root.path,
+        artifact.owner.workspace_root if artifact.owner else "",
+        artifact.owner.package if artifact.owner else "",
+    ):
+        if prefix:
+            prefix = prefix + "/"
+            if not path.startswith(prefix):
+                fail("Path %s doesn't start with %s" % (path, prefix))
+            path = path[len(prefix):]
+    return path
 
 def _get_strip(ctx):
-  strip = ctx.attr.strip_prefix
-  if strip:
-    if strip.startswith("/"):
-      _fail(ctx, "strip_prefix should not end with /")
-      strip = strip[1:]
-    if strip.endswith("/"):
-      _fail(ctx, "strip_prefix should not end with /")
-    else:
-      strip += "/"
-  return strip
+    strip = ctx.attr.strip_prefix
+    if strip:
+        if strip.startswith("/"):
+            _fail(ctx, "strip_prefix should not end with /")
```

Comment:
Pre-existing issue, but this should be "should not start with /"?

Reply:
I don't want to mix semantic changes into a cleanup cl. Please do the fixes as a followup.